validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -122,30 +120,29 @@ def __init__(self, completer, has_block, has_batch):
         self._has_block = has_block
         self._has_batch = has_batch
 
-    def handle(self, connection_id, message_content):
-        gossip_message = GossipMessage()
-        gossip_message.ParseFromString(message_content)
-        if gossip_message.content_type == gossip_message.BLOCK:
-            block = Block()
-            block.ParseFromString(gossip_message.content)
+    # pylint:disable=arguments-differ
+    def handle(self, connection_id, obj_tag_ttl):
```

Comment:
In a sense it doesn't implement the abstract base class correctly. The docstring for `Handler.handle` has 
```
        :param message_content: The bytes to be deserialized
                                into a protobuf python class
```
but this handler now expects something other than message content bytes. Leaving the argument as `message_content` would be misleading.

Reply:
I would agree.  Update the docstring to indicate that message content is either the raw bytes or a deserialized object. 

I would point out  that the name `obj_tag_ttl` provides very little indication as to what its value is, especially since you haven't documented its new meaning.